test/TestUtils/test_utils.cpp

```diff
@@ -126,7 +126,7 @@ GuidPrefix random_guid_prefix(
 Address random_address(
```

Comment:
The whole `test_utils` file is waiting for a refactor.
All the types uses very silly random systems, instead of generating real arbitrary values.
The idea of this function in the future is to create real arbitrary addresses.